src/v2/cookbook/packaing-sfc-for-npm.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Packaging Vue Components for npm
+type: cookbook
+order: 10
```

Comment:
We have order 10 and 11 now, this would need to be 12

Reply:
Will do.